cmd/iso8583/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+)
+
+var (
+	programName = filepath.Base(os.Args[0])
```

Comment:
I think `flag.Arg(0)` would be safer. It'll strip out the CLI arguments.

```
$ go doc flag arg
package flag // import "flag"

func Arg(i int) string
    Arg returns the i'th command-line argument. Arg(0) is the first remaining
    argument after flags have been processed. Arg returns an empty string if the
    requested element does not exist.

```

Reply:
I checked `flag.Arg(0)`

* It returns empty string if `flag.Parse()` was not called. We want to call `flag.Parse()` and use program name in `flag.Usage()` on error.
* According to the documentation, it returns first argument (command) when `flag.Parse()` was called:

```
➜ ./bin/iso8583 hello
flag.Arg(0) = hello
```